src/Microsoft.Health.Dicom.SqlServer/Registration/DicomServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -66,6 +66,10 @@ public static class DicomServerBuilderSqlServerRegistrationExtensions
                 .Scoped()
                 .AsImplementedInterfaces();
 
+            services.Add<SqlIndexDataStoreV3>()
+                .Scoped()
```

Comment:
nit: it's recommended to use `AsSelf()` before `AsImplementedIntefaces()` as [here](https://github.com/microsoft/healthcare-shared-components/blob/ab91e7a841b23b10ea5a611715528892d59fcdc1/src/Microsoft.Health.Extensions.DependencyInjection/TypeRegistrationBuilder.cs#L125)